CMakeLists.txt

```diff
@@ -340,6 +340,33 @@ install(FILES
     ${CMAKE_CURRENT_BINARY_DIR}/${PACKAGE_NAME}-config-version.cmake
   DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/${PACKAGE_NAME})
 
+# Create pkg-config file
+include(build/JoinPaths.cmake)
+# from: https://github.com/jtojnar/cmake-snips#concatenating-paths-when-building-pkg-config-files
+join_paths(DIRECTXTEX_INCLUDEDIR_FOR_PKG_CONFIG "\${prefix}" "${CMAKE_INSTALL_INCLUDEDIR}")
+join_paths(DIRECTXTEX_LIBDIR_FOR_PKG_CONFIG "\${prefix}"     "${CMAKE_INSTALL_LIBDIR}")
+
+set(DIRECTXTEX_DEP_L "")
+if(ENABLE_OPENEXR_SUPPORT)
+list(APPEND DIRECTXTEX_DEP_L "OpenEXR")
+list(APPEND DIRECTXTEX_DEP_L "zlib")
+endif()
+if(directxmath_FOUND)
```

Comment:
Shouldn't this be:

```
if(directxmath_FOUND)
    list(APPEND DIRECTXTEX_DEP_L "DirectXMath")
endif()
if(directx-headers_FOUND)
  list(APPEND DIRECTXTEX_DEP_L "DirectX-Headers")
endif()

Reply:
Your change requires this pull request I just made (https://github.com/microsoft/DirectXMath/pull/174 ).  I did not think DirectXMath needed a .pc file because it's a headers-only library so I did not make one earlier.